src/pages/ua/refugee-guide/{MarkdownRemark.frontmatter__slug}.js

```diff
@@ -9,12 +9,14 @@ import Faq from "../../../components/Faq";
 
 import Section from "../../../components/Section";
 
-export default function Template({ data }) {
+import { modifyCrumbs } from "../../../utils/modifyCrumbs";
```

Comment:
No need to split this out. It gets a bit more complex that way. I was hinting at using `useMemo` instead of regenerating the array each render (which it is still doing).